packages/scripts/scripts/env.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+const chalk = require( 'chalk' );
+
+/**
+ * Node dependencies.
+ */
+const { env, exit, stdout, cwd } = require( 'process' );
+const { normalize } = require( 'path' );
+const { existsSync } = require( 'fs' );
+
+/**
+ * Internal dependencies
+ */
+const {
+	getArgsFromCLI,
+	spawnScript,
+} = require( '../utils' );
+
+const args = getArgsFromCLI();
+
+if ( ! args.length ) {
+	const scripts = {
+		install: "If you don't have a check out of the WordPress repository that you want to work with, this will automatically download, configure, and connect to WordPress.",
```

Comment:
Do you plan to document this new group of scripts in the README file of `@wordpress/scripts` package as part of this PR?

Reply:
👍🏻 7fe5c3d7a